container/build/Containerfile

```diff
@@ -0,0 +1,115 @@
+###########################################################################
+## Production image #######################################################
+###########################################################################
+from public.ecr.aws/lts/ubuntu:20.04 as prod
```

Comment:
Bit of style bikeshedding here: Any particular reason for going with all lowercase on the instructions? As noted in https://docs.docker.com/reference/dockerfile/#format

> The instruction is not case-sensitive. However, convention is for them to be UPPERCASE to distinguish them from arguments more easily.

and my editor wants to flag every instruction in this file as an error to fix :sob: 

![Screenshot at 2025-01-08 15-35-34](https://github.com/user-attachments/assets/c0b6aac4-7a2e-49bc-a9e5-e5f87a101fd3)

Reply:
These lower case commands were solely personal preference: I prefer it when my config files don't shout at me.

I've switched them to upper case.